src/renderer/services/file-management-system/test/file-management-system.test.ts

```diff
@@ -2,23 +2,15 @@ import * as fs from "fs";
 import * as os from "os";
 import * as path from "path";
 
-import { expect } from "chai";
-import { noop } from "lodash";
-import { createSandbox, SinonStubbedInstance } from "sinon";
+import {expect} from "chai";
+import {noop} from "lodash";
+import {createSandbox, SinonStubbedInstance} from "sinon";
```

Comment:
Hmm this is not what the other files are formatted like so lets perhaps revert this unless we are committing to using this formatting

Reply:
reverted